src/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests/ProcessTestBase.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Diagnostics.ProcessTests
+{
+    public class ProcessTestBase : IDisposable
+    {
+        protected const int WaitInMS = 100 * 1000;
+        protected const string CoreRunName = "corerun";
+        protected const string TestExeName = "System.Diagnostics.Process.TestConsoleApp.exe";
+        protected const int SuccessExitCode = 100;
+
+        protected Process _process;
```

Comment:
On my box, they run for 6 sec. None of the tests actually Wait on a thread. We create processes that are supposed to run for infinite (100s) to ensure the process is alive, till we finish testing. Then we kill it in Dispose.